lib/src/widgets/chat/ds_message_bubble.widget.dart

```diff
@@ -30,7 +36,7 @@ class DSMessageBubble extends StatelessWidget {
       flex: 5,
       child: DSAnimatedSize(
         child: Container(
-          margin: EdgeInsets.fromLTRB(16, 2.0, 16, 2.0),
+          margin: const EdgeInsets.fromLTRB(16, 2.0, 16, 2.0),
```

Comment:
give preference to decimal use

Reply:
I believe that using symetrics can be more readable in this case.
          
          margin: const EdgeInsets.symmetric(
            horizontal: 16.0,
            vertical: 2.0,
          ),

Reply:
done